packages/components/organisms/f-header/src/assets/scss/navigation.scss

```diff
@@ -46,6 +46,9 @@ $nav-toggleIcon-color--transparent : $nav-icon-color--transparent;
 $nav-toggleIcon-bg                 : transparent;
 $nav-toggleIcon-space              : 5px;
 
+$nav-focus-borderRadius            : 800px;
```

Comment:
Not sure if this is the same as just using 50% as the `border-radius` potentially?

Reply:
ah I thought this too but turns out 50% creates a focus slightly different to the designs -
![image](https://user-images.githubusercontent.com/49618712/162235323-bc4774cb-8c3c-4933-a9ea-575bd0654835.png)